packages/cosmic-swingset/src/block-manager.js

```diff
@@ -79,6 +80,17 @@ export default function makeBlockManager({
         break;
       }
 
+      case ActionType.INSTALL_BUNDLE: {
+        p = (async () => {
+          const bundle = JSON.parse(action.bundle);
+          harden(bundle);
+          return validateAndInstallBundle(bundle);
+        })().catch(error => {
+          console.error(error);
```

Comment:
I imagine this error message would be opaque without some more context. Not sure what would be useful to the end user.

Reply:
yeah, it might be worthwhile logging the bundle ID that failed to install, so a validator operator could correlated it with the installation txn

and/or the "event" we emit with success/failure should include a copy of this `error` message

Reply:
I think this isn’t the right place to annotate the error with the bundle ID, though it already is annotated through many paths, and it wouldn’t hurt to make sure it’s on every pertinent one. I looked into this and ran into a chicken-and-egg with JSON.parse being a necessary step to see the bundle ID here.

Emitting an event here would be nice too, but I’m going to call it a future enhancement.